### DISCUSSION TITLE
Getting the env variables in custom plist file on iOS

### DISCUSSION THREAD

**Discussion by loabrantes** [maxkomarychev/react-native-ultimate-config#139] [Q&A]
Hello!

I'm trying to add an environment variable to a custom plist but without success.
My custom plist is about installing an SDK:

`ios/myProject/sdkName.plist`
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <!-->The Application ID<-->
    <key>applicationId</key>
     <string>$(MY_ID)</string>
    <!-->Indicates whether the SDK should present logs or not. Default value: true<-->
    <key>logEnabled</key>
    <true/>
    <!-->Indicates whether location tracking should be enabled by default during SDK initialization. Default value: true<-->
    <key>locationEnabledByDefault</key>
    <true/>
  </dict>
</plist>
```

As an attempt at a solution, I created this file:

`ios/sdkName.plist`
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
    <dict>
        <key>applicationId</key>
        <string>${MY_ID}</string>
    </dict>
</plist>
```
But also without success. Is there another way to do this? Am I doing something wrong?

> I'm using version `6.0.0`, my RN version is `0.71.8` and in my `Info.plist` file the variables work correctly.

**Comment by maxkomarychev** [comment]
Can you share more information about how these plist files are being consumed? Are they loaded in the code somehow?

Ar this moment I can speculate they are loaded at runtime and I am fairly sure for plist files other than info.plist interpolation of variables is not supported.
